hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/index/hbase/SparkHoodieHBaseIndex.java

```diff
@@ -385,7 +384,7 @@ private void doMutations(BufferedMutator mutator, List<Mutation> mutations, Rate
     mutations.clear();
   }
 
-  public Map<String, Integer> mapFileWithInsertsToUniquePartition(JavaRDD<WriteStatus> writeStatusRDD) {
+  Map<String, Integer> mapFileWithInsertsToUniquePartition(JavaRDD<WriteStatus> writeStatusRDD) {
```

Comment:
this method only used in test; changing to package access due to using RDD in the signature

Reply:
Since we don't want to depend on Guava we can add our own annotation with the same purpose as `@VisibleForTesting`